CHANGELOG.next.md

```diff
@@ -48,6 +48,37 @@ Thanks, you're awesome :-) -->
 
 #### Improvements
 
+#### Deprecated
+
+### Tooling and Artifact Changes
+
+#### Breaking changes
+
+#### Bugfixes
+
+#### Added
+
+#### Improvements
+
+#### Deprecated
+
+## 8.6.0 (Soft Feature Freeze)
```

Comment:
Looks like this got duplicated?

Also, should it be updated to HFF?

Reply:
Thanks Kylie, fixed